projects/ion/src/lib/popover/popover.directive.ts

```diff
@@ -35,6 +40,8 @@ export class IonPopoverDirective implements OnDestroy {
   @Input() ionPopoverPosition?: PopoverPosition = PopoverPosition.DEFAULT;
   @Input() ionPopoverArrowPointAtCenter = true;
   @Input() ionPopoverCustomClass?: string;
+  @Input() ionPopoverTrigger: PopoverProps['ionPopoverTrigger'] =
```

Comment:
What do you think about changing all of the inputs types to follow this pattern? Since you're already adding this one.

Reply:
I ended up doing it in the component too lol